src/videojs.ga.coffee

```diff
@@ -53,7 +55,7 @@ videojs.plugin 'ga', (options = {}) ->
         if "start" in eventsToTrack && percent == 0 && percentPlayed > 0
           sendbeacon( 'start', true )
         else if "percentsPlayed" in eventsToTrack && percentPlayed != 0
-          sendbeacon( 'percent played', true, percent )
+          sendbeacon( 'percent played'+(percent-percentsPlayedInterval)+"-"+percent, true, 1)
```

Comment:
I guess the idea is to have `20-30` instead of `20` in GA.
Changing the beacon from a number to a string could lead to break the plugin for some people.

Unfortunately I don't have much insights on how people are using this :-1: 
What do you think about adding this as an opt-in option?